iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -183,6 +183,11 @@ public void preCreateTable(org.apache.hadoop.hive.metastore.api.Table hmsTable)
     catalogProperties.put(InputFormatConfig.TABLE_SCHEMA, SchemaParser.toJson(schema));
     catalogProperties.put(InputFormatConfig.PARTITION_SPEC, PartitionSpecParser.toJson(spec));
     setCommonHmsTablePropertiesForIceberg(hmsTable);
+
+    hmsTable.getParameters().computeIfPresent(BaseMetastoreTableOperations.METADATA_LOCATION_PROP, (k, v) -> {
```

Comment:
is this equivalent with this:
```
if (hmsTable.getParameters().computeIfPresent(BaseMetastoreTableOperations.METADATA_LOCATION_PROP) != null) {
  hmsTable.getParameters().put(CREATE_HMS_TABLE_IN_HOOK, "true");
}
```

Reply:
Yes, you're right. Previously I had some code in the mapping function and I forgot the refactor it.